lua/pac3/editor/server/wear.lua

```diff
@@ -1,28 +1,46 @@
+local next = next
+local type = type
+local istable = istable
+local IsValid = IsValid
+local tostring = tostring
+local isfunction = isfunction
+local ProtectedCall = ProtectedCall
+
 pace.StreamQueue = pace.StreamQueue or {}
 
-local frame_number = 0
-local last_frame
-local ERROR_COLOR = Color(228, 37, 37)
+timer.Create("pac_check_stream_queue", 0.1, 0, function()
+	local item = table.remove(pace.StreamQueue)
+	if not item then return end
 
-local function catchError(err)
-	pac.Message(ERROR_COLOR, 'Error: ', err)
-	pac.Message(debug.traceback())
-end
+	local data = item.data
+	local filter = item.filter
+	local callback = item.callback
 
-timer.Create("pac_check_stream_queue", 0.1, 0, function()
-	if not pace.BusyStreaming and #pace.StreamQueue ~= 0 then
```

Comment:
Note, I removed `pace.BusyStreaming` because it doesn't seem to exist anywhere else